docker-compose.yml

```diff
@@ -167,6 +179,20 @@ services:
       ELASTICSEARCH_URL: http://elasticsearch:9200
     networks:
       - dev
+  presto:
+    build: presto
+    platform: linux/amd64
+    ports:
+      - "8000:8000"
+    volumes:
+      - "./presto:/app"
+    env_file:
+      - presto/.env_file
+    networks:
+      - dev
+    depends_on:
+      elasticmq:
+        condition: service_healthy
```

Comment:
Another problem I had on CI was race conditions between Presto and ElasticMQ. Waiting for the service to be up fixed the problem. It was not enough to add just `depends_on: elasticmq`... I had to add a proper healthcheck that waits for the HTTP service to be actually up.